.devcontainer/devcontainer.json

```diff
@@ -69,6 +69,15 @@
   "updateContentCommand": "bin/vscode-setup",
   "postCreateCommand": "echo 'source /usr/share/bash-completion/completions/git' >> ~/.bashrc",
 
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3390, 5432, 9000, 9999]
+  // Use 'forwardPorts' to make a list of ports inside the container available locally. The ports correspond to the following:
+  //   * 3390 - OIDC
+  //   * 4566 - localstack (email)
+  //   * 5432 - postgres
+  //   * 6645 - localstack (file uploads)
+  //   * 8888 - prod simulator
+  //   * 9000 - application
+  //   * 9100 - unit tests
+  //   * 9999 - browser tests
+  //   * 10000 - azureite
```

Comment:
```suggestion
  //   * 3390 - OIDC (dev)
  //   * 4566 - localstack (dev email & file upload)
  //   * 5432 - postgres (dev)
  //   * 6645 - localstack (browser-test email & file upload)
  //   * 8888 - prod simulator
  //   * 9000 - application (dev)
  //   * 9100 - application (unit tests)
  //   * 9999 - application (browser tests)
  //   * 10000 - azureite
```

Reply:
Adding more details to the descriptions

Reply:
Thank you! Updated - but put it in the portAttributes format.